src/app/features/cases/components/organisms/CaseTimeline/TimelineCaseEvent.tsx

```diff
@@ -88,18 +93,24 @@ const CaseEvent: React.FC<Props> = ({ caseEvent }) => {
       : 
       <DefinitionList key={ thread.id } thread="TODO" showDate={true} />
   )
-
+  const currentEvent = caseEvents[0]
```

Comment:
`currentEvent` can be undefined. This will throw a runtime error at line 100 and 101.